lib/core/request.c

```diff
@@ -713,19 +713,25 @@ void h2o_req_log_error(h2o_req_t *req, const char *module, const char *fmt, ...)
 
 #undef INITIAL_BUF_SIZE
 
+#define EMIT_ONE(input_, len_)                                                                                                     \
```

Comment:
When declaring a complicated macro, please
* use `do { ... } while (0)` so that the control flow would be preserved regardless of how the macro is used, and
* refrain from touching arguments multiple times, you can assign them to local variables within the do-while block, then use those local variables multiple times